cartographer_turtlebot2/package.xml

```diff
@@ -34,6 +34,9 @@
   <depend>depthimage_to_laserscan</depend>
   <depend>kobuki</depend>
   <depend>kobuki_core</depend>
+  <depend>message_runtime</depend>
   <depend>openni2_launch</depend>
+  <depend>roscpp</depend>
```

Comment:
Should we also add <build_depend>g++-static</build_depend> as in cartographer_ros?